websites/documentation/gatsby-config.mjs

```diff
@@ -4,7 +4,6 @@ import dotenv from 'dotenv';
 dotenv.config({ path: `.env.${process.env.NODE_ENV}` });
 const config = {
   flags: {
-    DEV_SSR: true,
```

Comment:
why remove the DEV_SSR?

Reply:
It's better to use the FAST_DEV flag. It includes the flags that we need
See here: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/flags.ts#L87